doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
the only reason I don't favor the title bar is 'cause I read [MRG+1] as 1 commit ahead of merged code, which wrong but a habit my brain won't break.